packages/features/ee/support/components/HelpMenuItem.tsx

```diff
@@ -186,6 +186,7 @@ export default function HelpMenuItem({ onHelpItemSelect }: HelpMenuItemProps) {
       <div className="text-subtle bg-muted w-full p-5">
         <p className="">{t("specific_issue")}</p>
         <button
+          // cspell:ignore defualt
```

Comment:
In some places, this `text-defualt` class is used. Can any one confirm whether it is intentional or bug?

For now, added `ignore` for safety.